cli/src/commands/file/track.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Jujutsu Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::io::Write;
+
+use jj_lib::working_copy::SnapshotOptions;
+use tracing::instrument;
+
+use crate::cli_util::CommandHelper;
+use crate::command_error::CommandError;
+use crate::ui::Ui;
+
+/// Start tracking specified paths in the working copy
+///
+/// Without arguments, all paths that are not ignored will be tracked.
+///
+/// By default, Jujutsu starts tracking all new files automatically. This
+/// command is not useful then. You can configure which paths to automatically
```

Comment:
Re "This command is not useful then", would

`jj file track ignored-file`

start tracking a file in `.gitignore`? I think it probably should, given that you can already end up tracking a `.gitignore`-d file if you edit `.gitignore`, but I'm not entirely sure.